en/docs/learn/authentication-session-persistence.md

```diff
@@ -160,7 +161,7 @@ mentioned above.
 |-----------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | enable_persistence                | This enables the persistence of session data. Therefore, this must be configured to `true` if you wish to enable session persistence.                                                                                                                                                                                          |
 | persist_temporary_data             | Setting this to `true` enables persistence of temporary caches that are created within an authentication request.                                                                                                                                                                                                              |
-| persistence_pool_size              | ‘persistence_pool_size’ equals to 0 means that it is disabled and then the authentication flow is blocked until this particular data persistence task is completed. To execute the persistence task in asynchronously, set the value to \>0 value. Based on the pool size, the system creates the task parallel to execute the persistence task that was in the queue. |
+| persistence_pool_size              | Please keep this value 0.  ‘persistence_pool_size’ equals to 0 means that it is disabled and then the authentication flow is blocked until this particular data persistence task is completed. To execute the persistence task in asynchronously, set the value to \>0 value. Based on the pool size, the system creates the task parallel to execute the persistence task that was in the queue. We have found that any value greater than 0, does not provide expected results, and sometimes cause unwanted side effects.. |
```

Comment:
To execute the persistence task asynchronously,

and sometimes causes. 

Also remove extra fullstop at the end.